hms-app/src/components/card-details/UserDetails.tsx

```diff
@@ -17,7 +17,7 @@ import { DELETE_BUTTON_COLOR, JOIN_BUTTON_COLOR } from '../../common/colors'
 
 type IProps = {
   userId: string
-  reloadList?: () => void
+  reloadList: () => void
```

Comment:
this could probably be renamed to `onUserDeleted` (that seems to be the only action that causes the callback to be triggered)

Reply:
if, however, it is used elsewhere then some other name would be appropriate.  The issue with `reloadList` as a name is that from within this file you don't know what "list" is, and list might not make any sense to a new component which will make use of the `UserDetails`